<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fileprodialog</class>
 <widget class="QDialog" name="Fileprodialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>293</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>542</width>
    <height>293</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>542</width>
    <height>293</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/images/app_theme.png</normaloff>:/images/images/app_theme.png</iconset>
  </property>
  <widget class="QComboBox" name="comboBox_usr_choose">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_indi_usr">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_indi_file">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_snd">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>发送文件</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/images/send_ico.png</normaloff>:/images/images/send_ico.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_usrico">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>32</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="src.qrc">:/images/images/administor.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_indi_size">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>大小：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filesize">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>210</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filename">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>32</width>
     <height>30</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/images/file_ico.png</normaloff>:/images/images/file_ico.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_indi_progress">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>进度条：</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
